e2e/e2etests/test_sui_withdraw_and_call_revert_with_call.go

```diff
@@ -24,10 +23,8 @@ func TestSuiWithdrawAndCallRevertWithCall(r *runner.E2ERunner, args []string) {
 	targetPackageID := r.SuiExample.PackageID.String()
 	amount := utils.ParseBigInt(r, args[0])
 
-	// create the payload for 'on_call' with invalid address
-	// taking the first 10 letters to form an invalid address
-	invalidAddress := sample.SuiAddress(r)[:10]
```

Comment:
I think it would be great to keep a E2E test with this case

Reply:
It's a good idea. https://github.com/zeta-chain/node/pull/3837/commits/738fbcd2685ed6ef81f1d12a0d880023d603e01c